startvs.cmd

```diff
@@ -14,10 +14,11 @@ SET PATH=%DOTNET_ROOT%;%PATH%
 SET sln=%1
 
 IF NOT EXIST %DOTNET_ROOT%\dotnet.exe (
```

Comment:
```suggestion
IF NOT EXIST "%DOTNET_ROOT%\dotnet.exe" (
```

As long as you're touching this file, this fixes problems for users with spaces in their source root.